gradle-conjure/src/test/groovy/com/palantir/gradle/conjure/ConjureJavaLocalCodegenPluginIntegrationSpec.groovy

```diff
@@ -180,15 +184,17 @@ class ConjureJavaLocalCodegenPluginIntegrationSpec extends IntegrationSpec {
         result.wasExecuted(':conjure-api:compileJava')
         result.wasExecuted(':conjure-api:generateConjure')
 
-        def recommendedProductDependencies = readRecommendedProductDeps(file('conjure-api/build/libs/conjure-api-1.0.0.jar'))
-        recommendedProductDependencies == '{"recommended-product-dependencies":[{' +
+        def expected = '{"recommended-product-dependencies":[{' +
                 '"product-group":"com.palantir.conjure",' +
                 '"product-name":"conjure",' +
                 '"minimum-version":"1.2.0",' +
                 '"recommended-version":"1.2.0",' +
                 '"maximum-version":"2.x.x",' +
                 '"optional":false' +
                 '}]}'
+        def jarFile = file('conjure-api/build/libs/conjure-api-1.0.0.jar')
+        readManifestRecommendedProductDeps(jarFile) == expected
+        readResourceRecommendedProductDeps(jarFile) == expected
```

Comment:
Bug is caught here! Previously, this successfully read a file with an empty array of dependencies....